static/js/reducers/enrollments_test.js

```diff
@@ -163,6 +168,36 @@ describe('enrollments', () => {
         });
       });
     });
+
+    it('should add a course enrollment successfully', () => {
+      addCourseEnrollmentStub.returns(Promise.resolve());
+
+      let courseKey = 'course_key';
+      return dispatchThen(addCourseEnrollment(courseKey), [
+        REQUEST_ADD_COURSE_ENROLLMENT,
+        RECEIVE_ADD_COURSE_ENROLLMENT_SUCCESS,
+      ]).then(state => {
+        assert.equal(state.coursePostStatus, FETCH_SUCCESS);
+        assert.ok(addCourseEnrollmentStub.calledWith(courseKey));
+        assert.ok(fetchCoursePricesStub.calledWith());
+        assert.ok(fetchDashboardStub.calledWith());
+      });
+    });
+
+    it('should fail to add a course enrollment', () => {
+      addCourseEnrollmentStub.returns(Promise.reject());
+
+      let courseKey = 'course_key';
+      return dispatchThen(addCourseEnrollment(courseKey), [
+        REQUEST_ADD_COURSE_ENROLLMENT,
+        RECEIVE_ADD_COURSE_ENROLLMENT_FAILURE,
+      ]).then(state => {
+        assert.equal(state.coursePostStatus, FETCH_FAILURE);
+        assert.ok(addCourseEnrollmentStub.calledWith(courseKey));
+        assert.notOk(fetchCoursePricesStub.calledWith());
+        assert.notOk(fetchDashboardStub.calledWith());
```

Comment:
`assert.isTrue` and `assert.isFalse` would be a lot more clear for the 3 assertions above and the ones at lines 181-183